framework/db/QueryBuilder.php

```diff
@@ -483,12 +486,25 @@ public function batchInsert($table, $columns, $rows, &$params = [])
             return '';
         }
 
+        return $this->constructInsertQueryString($table, $columns, $values);
+    }
+
+    /**
+     * Creates INSERT SQL statement
+     * @param $tableName
```

Comment:
Types should be specified, descriptions should be added. `@since` tag needs to be added.